_guides/managing-oauth-clients.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Managing OAuth Clients
+layout: article
+section: Developing Components
+order: 1
+since: 20190403
+---
+This document reveals the new approach to OAuth utilization within the platform
+and explains how to [manage](#managing-oauth-client) OAuth clients for
+components in a tenant.
+
+## Changes
+
+
+OAuth client was defined for a component in its environment variables. This way
+global access to the component could not be enabled, because users in other tenants would see the
+component and its keys. Now OAuth clients are defined
+separately from the components’ environment variables, allowing the users in the
+other tenant to see all available components without security breach.
+
+To manage OAuth clients the user requires the following permissions:
```

Comment:
Two things I'm asking myself:

* what does this list of permissions tell me? What am I supposed to do with it? Do I have them? If not, how do I get them?
* probably it's too early to mention them here?

Reply:
It says there that one should contact support, but I'll still move this a bit further